test/SecTester.Scan.Tests/SecTester.Scan.Tests.csproj

```diff
@@ -10,6 +10,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="System.Linq.Async" Version="6.0.1" />
```

Comment:
Do you use it somewhere?

Reply:
Use to test aync enumerable `<async enumerable>.LastOrDefaultAsync();`